statistics/handle/dump.go

```diff
@@ -154,29 +165,39 @@ func (h *Handle) DumpHistoricalStatsBySnapshot(dbName string, tableInfo *model.T
 	}()
 	pi := tableInfo.GetPartitionInfo()
 	if pi == nil {
-		return h.getTableHistoricalStatsToJSONWithFallback(dbName, tableInfo, tableInfo.ID, snapshot)
+		jt, fallback, err := h.getTableHistoricalStatsToJSONWithFallback(dbName, tableInfo, tableInfo.ID, snapshot)
+		if fallback {
+			fallbackTbls = append(fallbackTbls, tableInfo.Name.O)
```

Comment:
TableName only seems cannot identify a table, should we add DBName here as well?

Reply:
Yes!
Updated.